### PR TITLE
Fix link to homepage in composer.json

### PR DESCRIPTION
**Pull request by hannesvdvreken**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "league",
         "dic"
     ],
-    "homepage": "https://github.com/thephpleague/route",
+    "homepage": "https://github.com/thephpleague/dic",
```
Comment (GrahamCampbell) [inline composer.json]: ?

Comment (hannesvdvreken) [inline composer.json]: clearly this was a leftover after copying and changing https://github.com/orno/route/blob/league/composer.json

**Comment by philipobenito** [comment]
Correct :+1:

**Comment by GrahamCampbell** [comment]
But the link doesn't even work?

**Comment by hannesvdvreken** [comment]
Because the link points to a gh pages site which is not setup yet. It will be once this is repo is moved to the league gh organisation.

**Comment by philipobenito** [comment]
The code base has been moved over already as Orno\Route will need to stay here for current installs so it's actually fixed in there.

Just working on the best way to provide attribution for past contributions.

**Comment by hannesvdvreken** [comment]
Best way to keep attribution? "transfer" the repository and add a "[replace](https://getcomposer.org/doc/04-schema.md#replace)" to the `league/route` composer.json file.

Like `league/commonmark` (https://packagist.org/packages/league/commonmark and https://packagist.org/packages/colinodell/commonmark-php)

**Comment by philipobenito** [comment]
The problem here was that I wanted it to be v1 of the `League` version.

I know it's not the best way to proceed in the way of attribution, but the idea is that the Orno packages will stay in place for people to continue to use without forcing an immediate upgrade.

Apologies if it was the wrong way to go but I'll be providing as much transparency for contributions as I can.

**Comment by hannesvdvreken** [comment]
No problem ;-) that is your decision.

I'll be contributing to the `League` package from now on.

**Comment by philipobenito** [comment]
Thanks, it's appreciated :+1:

**Pull request merged**